test/test-generate-kmp.ts

```diff
@@ -0,0 +1,29 @@
+import { WordListFromArray } from "@predictive-text-studio/lexical-model-compiler";
+import { generateKmp } from "@worker/generate-kmp";
+
+import test from "ava";
+import JSZip = require("jszip");
+
+const langName = "English";
+const bcp47Tag = "en";
+const sources = [
+  new WordListFromArray("wordlist.xlsx", [
+    ["TŦE", 13644],
+    ["E", 9134],
+    ["SEN", 4816],
+  ]),
+];
+const modelID = "nrc.en.mtnt";
+
+test("it should generate a kmp file", async (t) => {
+  const kmp = await generateKmp(langName, bcp47Tag, sources, modelID);
```

Comment:
**[Question]**: is there a different way in Ava to test async function ?

Reply:
A different way how so? I'm having difficulting understanding exactly what you mean by this

Reply:
I was using Jasmine that provide the `fakeAsync ` for the use of testing async function. Since I am extremely new to AVA I wasn't sure if Ava has something similar, which is why I put it as a **[Question]**

Reply:
Ah, no, this is [the idiomatic way to write async tests](https://github.com/avajs/ava/blob/master/docs/01-writing-tests.md#async-function-support)!